specifyweb/frontend/js_src/lib/components/QueryBuilder/ToForms.tsx

```diff
@@ -34,18 +34,11 @@ export function QueryToForms({
       ? (results[index]![queryIdField] as number)
       : Array.from(selectedRows)[index];
 
-  const containUndefinedResults = React.useMemo(
-    () => results.includes(undefined),
-    [results]
-  );
-
   return (
     <>
       <Button.Small
         disabled={
-          results.length === 0 ||
-          totalCount === undefined ||
-          containUndefinedResults
+          results.length === 0 || totalCount === undefined || totalCount === 0
```

Comment:
`results.length === 0` and `totalCount === 0` seems redundant
is having just `totalCount === 0` sufficient?